### DISCUSSION TITLE
Etiketleme Ödülü

### DISCUSSION THREAD

**Discussion by shnkrcy** [IBAX-io/go-ibax#898] [Q&A]
### Describe the bug

Ödül kazanma kurallarında tweetlerin altına arkadaşlarını etiketle ve tweet için 200puan alıcaksın yazıyor fakat ben nerdeyse her tweetin altına etiketleme yapsamda puanımda hiç bir değişiklik olmadı.

### Reproduction

Bütün tweetlerin altında etiketleme yaptım hiç biri için ödül gelmedi.

### System Info

```shell
a
```

### Logs

```shell
a
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by hawkins09** [comment]
Aynen bende aynı sorunu yaşıyorum, bir çok tweetin altına etiketleme yaptım ama hiç puanıma yansımadı.
